src/hb-subset-plan.cc

```diff
@@ -558,6 +583,11 @@ hb_subset_plan_destroy (hb_subset_plan_t *plan)
   if (!hb_object_destroy (plan)) return;
 
   hb_set_destroy (plan->unicodes);
+  if (plan->unicode_to_new_gid_list)
+  {
+    plan->unicode_to_new_gid_list->fini ();
+    hb_free (plan->unicode_to_new_gid_list);
```

Comment:
Why use a pointer, instead of just a embedded vector?

Reply:
Being that hb_subset_plan is an hb_object so it's constructor/destructor doesn't get called I wasn't sure if it would be safe to use embedded vector. That and to keep consistency with the rest of the fields in subset plan.

Reply:
but now that I think about it more, this should be fine. I've updated to use embedded.

Reply:
Should also fix that memory fuzzer failure.